src/installer/test/HostActivation.Tests/FrameworkResolution/ResolvedFrameworksProperty.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit;
+
+namespace Microsoft.DotNet.CoreSetup.Test.HostActivation
+{
+    public class ResolvedFrameworksProperty : IClassFixture<ResolvedFrameworksProperty.SharedTestState>
+    {
+        private readonly SharedTestState sharedTestState;
+
+        public ResolvedFrameworksProperty(SharedTestState fixture)
+        {
+            sharedTestState = fixture;
+        }
+
+        [Fact]
+        public void ResolvedFrameworksPropertyIsEmptyForStandaloneApps()
+        {
+            var fixture = sharedTestState.StandaloneApp.Copy();
+            var dotnet = fixture.BuiltDotnet;
+            var appDll = fixture.TestProject.AppDll;
+
+            dotnet.Exec(appDll)
+                .EnableTracingAndCaptureOutputs()
+                .Execute()
+                .Should()
+                .Pass()
+                .And
+                .HaveStdErrContaining($"Property RESOLVED_FRAMEWORKS = \r\n");
+        }
+
+        [Fact]
+        public void ResolvedFrameworksPropertyIsComputedForPortableApps()
```

Comment:
Can you please add a test for cases where the requested version doesn't match the resolved version?
Ideally also a test with multiple resolved frameworks, but that is a bit harder to do.